darwin/client.py

```diff
@@ -5,19 +5,20 @@
 import zlib
```

Comment:
There are many changes in here which are just formatting or typing
* `Any` -> `UnknownType` are equivalent, but one doesn't cause issues with linter.
* Reformatting of lines has occurred in several unrelated places.

Only the implementation of the `_get_item_count` and its' use and testing are relevant to the fix